api/specs/web-server/_users.py

```diff
@@ -45,6 +50,25 @@ async def update_my_profile(_profile: ProfileUpdate):
     ...
 
 
+@router.get(
+    "/me/preferences",
+    response_model=Envelope[FrontendUserPreferencesGet],
```

Comment:
Just wondering here.
- you said you have now frontend, backend and services preferences right?
- the services preferences are a priori not available to the frontend (or maybe just as something that say whether one exists or not), what about the backend preferences?
--> if anything above is correct, then I guess what you return are just preferences not frontendpreferences type.
--> it is your responsibility to select what user preferences are going to the frontend